_includes/locations.html

```diff
@@ -0,0 +1,3 @@
+{% for location in event.locations %}
+    {{location}}{% unless forloop.last %} ·{% endunless -%}
```

Comment:
We currently rely on event editor to determine order of chapters, which makes most sense probably. We could add specific span tag/badges for each location, and make locations hyper-linkable to the individual chapter pages? 

When both online/global are present, we could hide them and display only "online"?